### ISSUE TITLE
Problem with the compiler

### ISSUE THREAD

**Issue by omar16100** [freeCodeCamp/freeCodeCamp#15152]
Orange color is shown as it is supposed to, but still i cant move to the next challange 

Challenge [override-class-declarations-by-styling-id-attributes](http://beta.freecodecamp.com/en/challenges/basic-css/override-class-declarations-by-styling-id-attributes) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64; rv:53.0) Gecko/20100101 Firefox/53.0</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

```html


<style>
  body {
    background-color: black;
    font-family: Monospace;
    color: green;
  }
  .pink-text {
    color: pink;
  }
  .blue-text {
    color: blue;
  }
  #orange-text {
  color: orange;
  }
</style>
<h1 id="orange-text" class="pink-text blue-text">Hello World!</h1>
```

![capture](https://cloud.githubusercontent.com/assets/23135110/26669717/7a15db88-46e1-11e7-8ff5-0476045d7c86.PNG)

**Comment by charlesBak** [comment]
hi @omar16100 i have tested your code in Chrome and it works fine. 
The problem seems to be with Firefox.

**Comment by jonathanihm** [comment]
I was also able to reproduce with Firefox 53.0.3

**Comment by JBallin** [comment]
Safari works

**Comment by omar16100** [comment]
Thank you!

**Comment by ArmoryCup** [comment]
I have same problem; it does not work in Chrome Version 58.0.3029.110 (64-bit) 
OS: Linux Mint 17.3 Cinnamon 64-bit

**Comment by lynxlynxlynx** [comment]
Probably also a duplicate of #15227, just with a different challenge.

**Comment by systimotic** [comment]
Thanks for looking into this! We recently fixed this (#15775), so I'm closing this issue.
If you think I'm wrong in closing this issue, please reopen it and add further clarification.

Happy coding!
